zipkin-lens/src/components/DiscoverPage/DiscoverPageContent.tsx

```diff
@@ -399,29 +404,60 @@ const DiscoverPageContent: React.FC<DiscoverPageContentProps> = ({
     }
   }, [setQueryParams, tempCriteria, tempLookback, tempLimit]);
 
-  const [traces, isLoadingTraces] = useSelector((state: RootState) => [
+  const [
+    traces,
+    isLoadingTraces,
+    traceSummaries,
+  ] = useSelector((state: RootState) => [
     state.traces.traces,
     state.traces.isLoading,
+    state.traces.traceSummaries as TraceSummary[],
```

Comment:
Is it possible to avoid this cast?

Reply:
Yes, but traceSummaries state is written in JavaScript, so this type is `any`..
So I want to cast until the type is attached to the state of traceSummaries..